golang/internal/helper/docker/network.go

```diff
@@ -0,0 +1,66 @@
+package docker
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+	"github.com/rs/zerolog/log"
+)
+
+func GetAllNetworks() ([]types.NetworkResource, error) {
+	ctx := context.Background()
```

Comment:
We get `ctx := context.Background()` from parameters in every function. Using the same way would be better.

Reply:
fixed